src/styles.ts

```diff
@@ -50,6 +50,7 @@ const widgetBubbleStyle = {
   bubbleLogo: {
     width: '40px',
     height: '40px',
```

Comment:
Todo: Need to check if we want to impose a restriction on both width and height. Restricting only one dimension should be sufficient.

Reply:
Height is not needed, remove it. [commit](https://github.com/arcana-network/auth/pull/76/commits/bff7fd91b5143e14443c3666bd63c056f9228e67)